code/modules/food_and_drinks/recipes/tablecraft/recipes_pastry.dm

```diff
@@ -399,6 +399,67 @@
 	result = /obj/item/food/donkpocket/gondola
 	subcategory = CAT_PASTRY
 
+/datum/crafting_recipe/food/donkpocket/donkmousebrown
```

Comment:
I don't really like having separate recipe per mice, our crafting menu is already not very great as is.

This should instead be a general recipe for any `/obj/item/food/deadmouse`, which creates a generic `/obj/item/food/donkpocket/deadmouse`. Then, *that* re-implements `/CheckParts`. That way, you can dynamically update the sprite depending on the mouse.

Search CheckParts across the codebase to find existing crafting recipes that do this.

Reply:
If you want to keep the different tastes and descriptions and whatnot, you still can with this, you should be able to replace the item in CheckParts.